index.bs

```diff
@@ -4031,37 +4031,59 @@ WebAuthn [=[RP]=].
 
 : Client extension input
 :: The Boolean value `true` to indicate that this extension is requested by the [=[RP]=].
-    <xmp class="idl">
-    partial dictionary AuthenticationExtensionsClientInputs {
-      boolean loc;
-    };
-    </xmp>
 
 : Client extension processing
 :: None, except creating the authenticator extension input from the client extension input.
 
 : Client extension output
-:: Returns a JavaScript object that encodes the location information in the authenticator extension output as a {{Coordinates}} value,
-    as defined by [[Geolocation-API]].
-    <xmp class="idl">
-    partial dictionary AuthenticationExtensionsClientOutputs {
-      Coordinates loc;
-    };
-    </xmp>
```

Comment:
This deletion is incorrect and would destroy the alignment with the W3C Geolocation API.  It must not happen.